migration/config/spire.spiffe.io_clusterspiffeids.yaml

```diff
@@ -0,0 +1,221 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Should this annotation be present?

Reply:
its auto generated so id rather not remove it.